packages/flutter/test/painting/edge_insets_test.dart

```diff
@@ -50,4 +50,25 @@ void main() {
     expect(EdgeInsets.lerp(null, b, 0.25), equals(b * 0.25));
     expect(EdgeInsets.lerp(a, null, 0.25), equals(a * 0.75));
   });
+
+  test('EdgeInsets.resolve()', () {
+    expect(new EdgeInsetsDirectional.fromSTEB(10.0, 20.0, 30.0, 40.0).resolve(TextDirection.ltr), const EdgeInsets.fromLTRB(10.0, 20.0, 30.0, 40.0));
+    expect(new EdgeInsetsDirectional.fromSTEB(99.0, 98.0, 97.0, 96.0).resolve(TextDirection.rtl), const EdgeInsets.fromLTRB(97.0, 98.0, 99.0, 96.0));
```

Comment:
Well since we decided long ago never to do vertical text, the B and A are always top and bottom respectively. So the question is, is there a benefit to going BASE? I'm not convinced people really know what it means, so the win of using "common" terminology is minimal IMHO. The loss is that people have to learn new terms for top and bottom which aren't very intuitive (before and after what? the line? how is that different from start and end? etc).